src/format-error.ts

```diff
@@ -9,14 +9,15 @@ type Formatter = (m: string) => string;
 
 export function createFormatError(
 	bundlerMap: BundlerMap,
+	basePath: string,
 	formatError?: Formatter,
 ) {
 	const consumers = new WeakMap<RawSourceMap, SourceMapConsumer>();
-	const regex = /((?:\b[A-Z]:)?\/[^ #?:]+)[^ :]*:(\d+):(\d+)/gi;
-	//             |            |          ||    ||    |^^^^^^ Column
-	//             |            |          ||    |^^^^^^ Line
-	//             |            |          |^^^^^^ Eat any reamining URL (query in particular)
-	//             ^^^^^^^^^^^^^^^^^^^^^^^^^ URL pathname extraction
+	const regex = /((?:\b[A-Z]:|[^ #?:(]+)?\/[^ #?:]+)[^ :]*:(\d+):(\d+)/gi;
```

Comment:
I think this is just preventing the opening `(` from being captured as part of the URL? If so, that could just modify the existing regex:

```suggestion
	const regex = /((?:\b[A-Z]:)?[^ #?:(]+)[^ :]*:(\d+):(\d+)/gi;
```